### DISCUSSION TITLE
Issues with charging

### DISCUSSION THREAD

**Discussion by Captain0Kangaroo** [nathanmarlor/foxess_em#223] [General]
Firstly I'd like to say i really like the idea of this but for the life of me just cant get it working right.

Been installed for months and have updated when available but every time i try and use it i either end up with a battery not charged (which is the case this morning) or the battery charges but the Min SoC is set to a high value so i'm effectively running on grid.

This morning my battery is 10%. I checked the fox settings and force charge is 00:30-04:30 and both tabs are enabled. Min OoC are 10% for both tabs.

What am i doing wrong?

**Comment by nathanmarlor** [comment]
Can you include some logs please? Are you using the Fox cloud integration? There's a new modbus version which is much more stable.

**Reply by Captain0Kangaroo** [reply]
I'm using the Modbus version now. I was using cloud before but switched.
Whats the best way to export logs?

**Reply by nathanmarlor** [reply]
If you head to settings -> system -> logs and download from there please.

Random thought - you're not restarting HA overnight by chance? It needs to run consistently over the charge period.

**Reply by Captain0Kangaroo** [reply]
That might be it... I did a reboot last night at about 1am (to updated the new version). 
The logs have everything  - Is there w way to filter them for just the EM?

**Reply by nathanmarlor** [reply]
If you've rebooted it won't have 'cleaned up' after it started charging so expect your charging parameters to be in an odd state (i.e. min SoC, on/off). 

Can you set debug logging on and try again tonight please?
